@shared/api/internal.ts

```diff
@@ -960,6 +1024,7 @@ export const handleSignedHwPayload = async ({
 export const signTransaction = async (): Promise<void> => {
   try {
     await sendMessageToBackground({
+      activePublicKey: null,
       type: SERVICE_TYPES.SIGN_TRANSACTION,
     });
```

Comment:
I'm just trying to understand it a little better, would you mind explaining why we should omit `activePublicKey` in `SIGN_TRANSACTION` but is forwarded on `SIGN_FREIGHTER_TRANSACTION`? Thanks!

Reply:
I see the other Freighter API methods like `SIGN_BLOB` and `SIGN_AUTH_ENTRY` also don't take it probably under the same reason, but it's not 100% clear to me why that would be the case - thanks!

Reply:
Oh actually - this is a great callout! I was under the impression that because we have an account picker on the Sign Transaction/Blob/Auth entry screen, this wouldn't be needed. But upon further inspection, this _is_ needed. I'm adding this now!